rpcs3/Emu/Cell/Modules/cellGem.cpp

```diff
@@ -645,14 +751,15 @@ error_code cellGemGetInertialState(u32 gem_num, u32 state_flag, u64 timestamp, v
 	}
 
 	if (g_cfg.io.move == move_handler::fake)
-	{
-		map_to_ds3_input(gem_num, inertial_state->pad.digitalbuttons, inertial_state->pad.analog_T);
-		map_ext_to_ds3_input(gem_num, inertial_state->ext);
+		ds3_input_to_pad(gem_num, inertial_state->pad.digitalbuttons, inertial_state->pad.analog_T);
+	else if (g_cfg.io.move == move_handler::mouse)
+		mouse_input_to_pad(gem_num, inertial_state->pad.digitalbuttons, inertial_state->pad.analog_T);
 
-		inertial_state->timestamp = gem->timer.GetElapsedTimeInMicroSec();
+	if (!(g_cfg.io.move == move_handler::null)) {
```

Comment:
Remove this check and always execute the code below.
Move the nulll handler check above the other move handler checks and return CELL_OK if move_handler::null

Reply:
same, on master is same only for fake, so here i just set fake/mouse get same value